### ISSUE TITLE
lxml wheel fails, tries to use libxml2 2.9.3, brew has installed 2.9.4

### ISSUE THREAD

**Issue by jondelmil** [getnikola/nikola#2458]
Hi, I'm trying to install Nikola on MacOS 10.11 w/ python3.5. I've already followed the advice in previous issues directing users to

```
brew install libxml2 && brew link libxml2 --force
brew install libxslt && brew link libxslt --force
```

Here's my error:

```
pip install --upgrade "Nikola[extras]"
Collecting Nikola[extras]
  Using cached Nikola-7.7.12-py2.py3-none-any.whl
Requirement already up-to-date: python-dateutil>=2.4.0 in /Users/jonmiller/.virtualenvs/jondelmil.github.io/lib/python3.5/site-packages (from Nikola[extras])
Requirement already up-to-date: setuptools>=5.4.1 in /Users/jonmiller/.virtualenvs/jondelmil.github.io/lib/python3.5/site-packages (from Nikola[extras])

...

Collecting ptyprocess (from terminado>=0.3.3; sys_platform != "win32"->notebook>=4.0.0; extra == "extras"->Nikola[extras])
  Using cached ptyprocess-0.5.1-py2.py3-none-any.whl
Collecting wcwidth (from prompt-toolkit<2.0.0,>=1.0.3->ipython>=4.0.0->ipykernel>=4.0.0; extra == "extras"->Nikola[extras])
  Using cached wcwidth-0.1.7-py2.py3-none-any.whl
Building wheels for collected packages: lxml
  Running setup.py bdist_wheel for lxml ... -

Building wheels for collected packages: lxml
  Running setup.py bdist_wheel for lxml ... error
  Complete output from command /Users/jonmiller/.virtualenvs/jondelmil.github.io/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-build-6jeejm59/lxml/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/tmp64wukgz2pip-wheel- --python-tag cp35:
  Building lxml version 3.6.3.
  Building without Cython.
  Using build configuration of libxslt 1.1.28
  Building against libxml2/libxslt in one of the following directories:
    /usr/local/Cellar/libxslt/1.1.28_1/lib
    /usr/local/Cellar/libxml2/2.9.3/lib
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.11-x86_64-3.5
  creating build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/_elementpath.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/cssselect.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/doctestcompare.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/ElementInclude.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/pyclasslookup.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/sax.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/usedoctest.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/_diffcommand.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/_html5builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/_setmixin.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/clean.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/defs.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/diff.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/ElementSoup.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/formfill.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/html5parser.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/soupparser.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/usedoctest.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron
  copying src/lxml/isoschematron/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron
  copying src/lxml/lxml.etree.h -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/lxml.etree_api.h -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/includes/c14n.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/config.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/dtdvalid.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/etreepublic.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/htmlparser.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/relaxng.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/schematron.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/tree.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/uri.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xinclude.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xmlerror.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xmlparser.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xmlschema.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xpath.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xslt.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/etree_defs.h -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/lxml-version.h -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/rng
  copying src/lxml/isoschematron/resources/rng/iso-schematron.rng -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/rng
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
  copying src/lxml/isoschematron/resources/xsl/RNG2Schtrn.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
  copying src/lxml/isoschematron/resources/xsl/XSD2Schtrn.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_abstract_expand.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_dsdl_include.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_message.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_skeleton_for_xslt1.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_svrl_for_xslt1.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  running build_ext
  building 'lxml.etree' extension
  creating build/temp.macosx-10.11-x86_64-3.5
  creating build/temp.macosx-10.11-x86_64-3.5/src
  creating build/temp.macosx-10.11-x86_64-3.5/src/lxml
  clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/Cellar/libxslt/1.1.28_1/include -I/usr/local/Cellar/libxml2/2.9.4/include/libxml2 -Isrc/lxml/includes -I/usr/local/Cellar/python3/3.5.2_1/Frameworks/Python.framework/Versions/3.5/include/python3.5m -c src/lxml/lxml.etree.c -o build/temp.macosx-10.11-x86_64-3.5/src/lxml/lxml.etree.o -w -flat_namespace
  clang -bundle -undefined dynamic_lookup build/temp.macosx-10.11-x86_64-3.5/src/lxml/lxml.etree.o -L/usr/local/Cellar/libxslt/1.1.28_1/lib -L/usr/local/Cellar/libxml2/2.9.3/lib -lxslt -lexslt -lrt -lxml2 -lz -lm -o build/lib.macosx-10.11-x86_64-3.5/lxml/etree.cpython-35m-darwin.so
  ld: warning: directory not found for option '-L/usr/local/Cellar/libxml2/2.9.3/lib'
  ld: library not found for -lrt
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
  error: command 'clang' failed with exit status 1

  ----------------------------------------
```

Any ideas? There seems to be a discrepancy between libxml2 v2.9.3 and v2.9.4, but I'm not sure how to proceed. I've installed 2.9.4 through brew.

**Comment by ralsina** [comment]
All Nikola does is basically call "pip install lxml".

Try doing that yourself, if that fails, then it's a lxml bug. If that works, then we'll see what we can do.

**Comment by Kwpolska** [comment]
It's possible lxml depends on that older version, but would it really know
Homebrew directory structures? Does the 2.9.3 directory exist? Any links to
it in /usr/local/lib or elsewhere?

Also, I didn't need to install libxml manually (it's possible it was a
dependency of something else and I just got lucky).

-- 
Chris Warrick <https://chriswarrick.com/>

**Comment by jondelmil** [comment]
@Kwpolska No, there's no 2.9.3 directory. All libxml2 dylibs in /usr/local/lib/ point to the 2.9.4 directory, I don't know where else to look for other links.

@ralsina `pip isntall lxml` does fail. Looks like it's using an older build configuration from libxslt 1.1.28 but I'm not sure how to change that! Here's the output:

```
$ pip install lxml
Collecting lxml
  Using cached lxml-3.6.3.tar.gz
Building wheels for collected packages: lxml
  Running setup.py bdist_wheel for lxml ... error
  Complete output from command /Users/jonmiller/.virtualenvs/jondelmil.github.io/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-build-76c3rsg7/lxml/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/tmpyf6jm9lzpip-wheel- --python-tag cp35:
  Building lxml version 3.6.3.
  Building without Cython.
  Using build configuration of libxslt 1.1.28
  Building against libxml2/libxslt in one of the following directories:
    /usr/local/Cellar/libxslt/1.1.28_1/lib
    /usr/local/Cellar/libxml2/2.9.3/lib
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.11-x86_64-3.5
  creating build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/_elementpath.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/cssselect.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/doctestcompare.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/ElementInclude.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/pyclasslookup.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/sax.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/usedoctest.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/_diffcommand.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/_html5builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/_setmixin.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/clean.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/defs.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/diff.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/ElementSoup.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/formfill.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/html5parser.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/soupparser.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  copying src/lxml/html/usedoctest.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron
  copying src/lxml/isoschematron/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron
  copying src/lxml/lxml.etree.h -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/lxml.etree_api.h -> build/lib.macosx-10.11-x86_64-3.5/lxml
  copying src/lxml/includes/c14n.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/config.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/dtdvalid.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/etreepublic.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/htmlparser.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/relaxng.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/schematron.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/tree.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/uri.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xinclude.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xmlerror.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xmlparser.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xmlschema.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xpath.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/xslt.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/etree_defs.h -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  copying src/lxml/includes/lxml-version.h -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/rng
  copying src/lxml/isoschematron/resources/rng/iso-schematron.rng -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/rng
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
  copying src/lxml/isoschematron/resources/xsl/RNG2Schtrn.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
  copying src/lxml/isoschematron/resources/xsl/XSD2Schtrn.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
  creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_abstract_expand.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_dsdl_include.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_message.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_skeleton_for_xslt1.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_svrl_for_xslt1.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  running build_ext
  building 'lxml.etree' extension
  creating build/temp.macosx-10.11-x86_64-3.5
  creating build/temp.macosx-10.11-x86_64-3.5/src
  creating build/temp.macosx-10.11-x86_64-3.5/src/lxml
  clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/Cellar/libxslt/1.1.28_1/include -I/usr/local/Cellar/libxml2/2.9.4/include/libxml2 -Isrc/lxml/includes -I/usr/local/Cellar/python3/3.5.2_1/Frameworks/Python.framework/Versions/3.5/include/python3.5m -c src/lxml/lxml.etree.c -o build/temp.macosx-10.11-x86_64-3.5/src/lxml/lxml.etree.o -w -flat_namespace
  clang -bundle -undefined dynamic_lookup build/temp.macosx-10.11-x86_64-3.5/src/lxml/lxml.etree.o -L/usr/local/Cellar/libxslt/1.1.28_1/lib -L/usr/local/Cellar/libxml2/2.9.3/lib -lxslt -lexslt -lrt -lxml2 -lz -lm -o build/lib.macosx-10.11-x86_64-3.5/lxml/etree.cpython-35m-darwin.so
  ld: warning: directory not found for option '-L/usr/local/Cellar/libxml2/2.9.3/lib'
  ld: library not found for -lrt
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
  error: command 'clang' failed with exit status 1

  ----------------------------------------
  Failed building wheel for lxml
  Running setup.py clean for lxml
Failed to build lxml
Installing collected packages: lxml
  Running setup.py install for lxml ... error
    Complete output from command /Users/jonmiller/.virtualenvs/jondelmil.github.io/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-build-76c3rsg7/lxml/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-2ur0ay5n-record/install-record.txt --single-version-externally-managed --compile --install-headers /Users/jonmiller/.virtualenvs/jondelmil.github.io/bin/../include/site/python3.5/lxml:
    Building lxml version 3.6.3.
    Building without Cython.
    Using build configuration of libxslt 1.1.28
    Building against libxml2/libxslt in one of the following directories:
      /usr/local/Cellar/libxslt/1.1.28_1/lib
      /usr/local/Cellar/libxml2/2.9.3/lib
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.11-x86_64-3.5
    creating build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/_elementpath.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/cssselect.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/doctestcompare.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/ElementInclude.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/pyclasslookup.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/sax.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/usedoctest.py -> build/lib.macosx-10.11-x86_64-3.5/lxml
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/_diffcommand.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/_html5builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/_setmixin.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/builder.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/clean.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/defs.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/diff.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/ElementSoup.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/formfill.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/html5parser.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/soupparser.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    copying src/lxml/html/usedoctest.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/html
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron
    copying src/lxml/isoschematron/__init__.py -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron
    copying src/lxml/lxml.etree.h -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/lxml.etree_api.h -> build/lib.macosx-10.11-x86_64-3.5/lxml
    copying src/lxml/includes/c14n.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/config.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/dtdvalid.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/etreepublic.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/htmlparser.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/relaxng.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/schematron.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/tree.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/uri.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/xinclude.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/xmlerror.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/xmlparser.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/xmlschema.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/xpath.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/xslt.pxd -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/etree_defs.h -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    copying src/lxml/includes/lxml-version.h -> build/lib.macosx-10.11-x86_64-3.5/lxml/includes
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/rng
    copying src/lxml/isoschematron/resources/rng/iso-schematron.rng -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/rng
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
    copying src/lxml/isoschematron/resources/xsl/RNG2Schtrn.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
    copying src/lxml/isoschematron/resources/xsl/XSD2Schtrn.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl
    creating build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_abstract_expand.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_dsdl_include.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_message.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_skeleton_for_xslt1.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_svrl_for_xslt1.xsl -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.macosx-10.11-x86_64-3.5/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
    running build_ext
    building 'lxml.etree' extension
    creating build/temp.macosx-10.11-x86_64-3.5
    creating build/temp.macosx-10.11-x86_64-3.5/src
    creating build/temp.macosx-10.11-x86_64-3.5/src/lxml
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/Cellar/libxslt/1.1.28_1/include -I/usr/local/Cellar/libxml2/2.9.4/include/libxml2 -Isrc/lxml/includes -I/usr/local/Cellar/python3/3.5.2_1/Frameworks/Python.framework/Versions/3.5/include/python3.5m -c src/lxml/lxml.etree.c -o build/temp.macosx-10.11-x86_64-3.5/src/lxml/lxml.etree.o -w -flat_namespace
    clang -bundle -undefined dynamic_lookup build/temp.macosx-10.11-x86_64-3.5/src/lxml/lxml.etree.o -L/usr/local/Cellar/libxslt/1.1.28_1/lib -L/usr/local/Cellar/libxml2/2.9.3/lib -lxslt -lexslt -lrt -lxml2 -lz -lm -o build/lib.macosx-10.11-x86_64-3.5/lxml/etree.cpython-35m-darwin.so
    ld: warning: directory not found for option '-L/usr/local/Cellar/libxml2/2.9.3/lib'
    ld: library not found for -lrt
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    error: command 'clang' failed with exit status 1

    ----------------------------------------
Command "/Users/jonmiller/.virtualenvs/jondelmil.github.io/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-build-76c3rsg7/lxml/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-2ur0ay5n-record/install-record.txt --single-version-externally-managed --compile --install-headers /Users/jonmiller/.virtualenvs/jondelmil.github.io/bin/../include/site/python3.5/lxml" failed with error code 1 in /private/var/folders/d6/mncgl4sj6p36t84x96dx1wtc0000gn/T/pip-build-76c3rsg7/lxml/
```

**Comment by ralsina** [comment]
@jondelmil I'm afraid you need someone that actually knows about brew :-P

**Comment by Kwpolska** [comment]
On Fri, Aug 19, 2016 at 01:09:01PM -0700, Roberto Alsina wrote:
> @jondelmil I'm afraid you need someone that actually knows about brew :-P

I'll take a look at it on Monday.

-- 
Chris Warrick <https://chriswarrick.com/>
PGP: 5EAAEA16

**Comment by Kwpolska** [comment]
This is a lxml bug, fixed in 3.6.4 (try installing again). Also, you don’t need (and generally, don’t want) those two packages:

    brew unlink libxml2 libxslt
    brew uninistall libxml2 libxslt  # if you installed this for Nikola only

**Comment by jondelmil** [comment]
Thank you!
